client/db_test.go

```diff
@@ -288,3 +290,46 @@ func TestDebugName(t *testing.T) {
 		return nil
 	})
 }
+
+func TestCommonMethods(t *testing.T) {
+	types := []reflect.Type{
+		reflect.TypeOf(&client.DB{}),
+		reflect.TypeOf(&client.Tx{}),
+		reflect.TypeOf(&client.Batch{}),
+		reflect.TypeOf(client.DB{}.B),
+	}
+	blacklist := map[string]bool{
+		"AdminMerge":          true,
+		"AdminSplit":          true,
+		"Commit":              true,
+		"DebugName":           true,
+		"InternalAddCall":     true,
+		"InternalKV":          true,
+		"InternalSetPriority": true,
+		"Run":                  true,
+		"SetDebugName":         true,
+		"SetSnapshotIsolation": true,
+		"Tx": true,
+	}
+	for _, typ := range types {
+		for j := 0; j < typ.NumMethod(); j++ {
+			m := typ.Method(j)
+			var ch rune
+			for _, ch = range m.Name {
```

Comment:
Yes. It decodes the whole string (and allocates for the result) so it's a bit less efficient than utf8.DecodeRune, but it's probably the simplest option and performance doesn't matter here.